op-gateway/src/worker.ts

```diff
@@ -0,0 +1,43 @@
+import { Server } from '@ensdomains/ccip-read-cf-worker';
+interface Env {
+    L1_PROVIDER_URL: string
+    L2_PROVIDER_URL: string
+    L2_OUTPUT_ORACLE: string
+    DELAY: number
+}
+interface Router{
+  handle:(request:Request)=> void
+}
+let app:Router  
+async function fetch(request:Request, env:Env){
```

Comment:
Could we provide some kind of utility function on `evm-gateway` for this, to reduce the boilerplate?